packages/Molecules/Search/SearchForm.vue

```diff
@@ -4,22 +4,22 @@
     <h3 v-if="subtitle">{{ subtitle }}</h3>
     <div class="rpl-search-form__field" v-if="textSearch">
       <label>
-        <span class="rpl-search-form__label-text">Search for</span>
+        <span class="rpl-search-form__label-text">{{ searchInputLabel }}</span>
         <input v-model="searchInput" ref="searchinput" class="rpl-search-form__input" type="text" :placeholder="searchPlaceholder" @keydown.enter="submitSearch()" />
       </label>
       <button @click="submitSearch()" class="rpl-search-form__btn">
-        <span>Search</span>
+        <span :class="{'rpl-visually-hidden' : buttonHiddenLabel}">{{ buttonLabel }}</span>
         <rpl-icon symbol="search" :color="(theme === 'dark') ? 'white' : 'primary'" size="l"></rpl-icon>
       </button>
     </div>
     <button
-      v-if="filterForm && !expandFilters"
+      v-if="filterForm && showExpandButton"
```

Comment:
Hi @alan-cole if we don't need this anymore, we should reverse it back. I prefer to keep it simple. Cheers.

Reply:
Removed.